### ISSUE TITLE
Compile error introduced with 0.19.0

### ISSUE THREAD

**Issue by DanEastBentley** [iTwin/iTwinUI#142]
#### Environment
- "@itwin/itwinui-css": "^0.19.0",
- Windows 10
- Chrome 91.0.4472.77

#### Steps to reproduce
Upgrade to itwinui-css 0.19.0

#### Actual behavior
Getting build error:
```
Error: Undefined variable.
    ╷
    │ $iui-component-padding-horizontal: $iui-sm; // 12px
    │                                    ^^^^^^^
    ╵
```

#### Expected behavior
No build errors.

**Comment by mayank99** [comment]
We just released 0.20.0. Can you check if that fixes your build error?

**Comment by DanEastBentley** [comment]
Yes, the build error has been fixed. Thanks.
